core/ecschema-editing/src/Editing/ECClasses.ts

```diff
@@ -28,6 +29,23 @@ export class ECClasses {
 
   protected constructor(protected _schemaEditor: SchemaContextEditor) { }
 
+  /**
+   * Allows access for editing of base Property attributes.
+   */
+  public readonly properties = new Properties(this._schemaEditor);
```

Comment:
Since enumerationProperties and primitiveProperties also rather act as placeholder. I wonder if we should not also add placeholders for the missing navigation- and struct- properties. Then we would not need to make the base Properties public, as this is all covered through inheritance. 

And, when overriding the getProperty method, we can even ensure we get the correct property type in the editor call.

Reply:
OK. I will add them.

Reply:
If all property editor classes derive from Properties editor, there is no need to keep line 33, right?

Reply:
based on our conversation, can you resolve this?